features/font-loading.yml

```
@@ -0,0 +1,47 @@
+name: Font loading
+description: The CSS Font Loading API dynamically loads custom fonts and tracks when fonts are loading or ready to use.
```

I'm going to apply this to be able to merge today, but for posterity:

Prefer an essential entry point to an API over a spec name. It's much more concrete and directly connects with what a developer does with this feature.

```suggestion
description: The `document.fonts` API dynamically loads custom fonts and tracks when fonts are loading or ready to use.
```